Integrations/MimecastV2/MimecastV2.yml

```diff
@@ -32,8 +32,8 @@ configuration:
   name: secretKey
   required: false
   type: 4
-- defaultvalue: 'true'
-  display: Trust any certificate (unsecure)
+- defaultvalue: ''
```

Comment:
revert

Reply:
are we still reverting this? if so we need to cancel the hook + tell everyone who changed the default value according to hooks to revert as well.